### ISSUE TITLE
could you please provide me the weight of VideoMAEv2 pre-trained on Kinetics-400?

### ISSUE THREAD

**Issue by Value-Jack** [OpenGVLab/VideoMAEv2#3]
I want to use the the weight to extract the features of the thumos14.
I'll  appreciate it very much  if you could share me the weights or **the features of the thumos14** extracted by Kinetics-400  ！

**Comment by congee524** [comment]
Hi, the weights are available at [ModelZoo](https://github.com/OpenGVLab/VideoMAEv2/blob/master/docs/MODEL_ZOO.md)

**Comment by fmthoker** [comment]
@wanglimin  The shared models do not contain any model with Kinetics-400 pretraining. Can you share a ViT-B model pre-trained on Kinetics-400 with dual masking?  I am doing a  comparison of Video SSL models and want to include Videomae and Videomaev2 with the same backbone and pretraining dataset.
